src/index.html

```diff
@@ -7,7 +7,72 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="stars stars--0">
+      <div class="stars--star"></div>
+
+      <div class="stars--star"></div>
+      
+      <div class="stars--star"></div>
+      
+      <div class="stars--star"></div>
+      
+      <div class="stars--star"></div>
+    </div>
```

Comment:
stars is block, star is element, stars--0 is modifier (BEM)

```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```

Reply:
also remove empty lines between empty divs